src/index.html

```diff
@@ -4,19 +4,122 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, 
+      user-scalable=no, 
+      initial-scale=1.0, 
+      maximum-scale=1.0, 
+      minimum-scale=1.0"
     />
     <meta
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="use-credentials"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+          src="images/logo.png"
+          class="logo__img"
+          alt="logo"
```

Comment:
```suggestion
          alt="MOYO header logo"
```
<img width="549" alt="Знімок екрана 2024-08-13 о 18 33 09" src="https://github.com/user-attachments/assets/cc972165-0a9a-40d5-929c-1a1d4e086421">